File: jsanchez_it/Sol_Jose_8_1/Ejemplos Patrones/5 WPF con EF Repos Dependency Injection/Academy/Academy.Lib/Models/Student.cs
using Common.Lib.Core;

namespace Academy.Lib.Models
{
    public class Student : Entity
    {
        public string Name { get; set; }
        public string Email { get; set; }
        public string Dni { get; set; }

        public int ChairNumber { get; set; }

        public Student()
        {

        }


    }
}

File: jsanchez_it/AcademyFinal - asp/AcademyFinal/Academy.Web/wwwroot/js/App/Models/Student.js

class Student {

    get Name() {
        return this._name;
    }

    set Name(value) {
        this._name = value;
    }


    get Dni() {
        return this._dni;
    }

    set Dni(value) {
        this._dni = value;
    }


    get ChairNumber() {
        return this._chairNumber;
    }

    set ChairNumber(value) {
        this._chairNumber = value;
    }


    get Email() {
        return this._email;
    }

    set Email(value) {
        this._email = value;
    }

    constructor(name, dni, chairNumber, email) {
        this.Name = name;
        this.Dni = dni;
        this.ChairNumber = chairNumber;
        this.Email = email;

    }


}



